packages/devtools_server/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: devtools_server
 description: A server that supports Dart DevTools.
 
-version: 0.1.13-dev
+version: 0.1.13
```

Comment:
Were we depending on some unreleased feature in the server? Do we need to release devtools_server first?

Reply:
I don't know that is my concern that devtools' dependency is today on 0.1.12.  So I'm okay with leaving it this way for 0.1.14 and we figure if we should migrate to 0.1.13 later.

Reply:
sg to leave as is for 0.1.14. I filed an [issue](https://github.com/flutter/devtools/issues/1598) to investigate this for the next release

Reply:
The change we haven't really tested is the server using the latest sse package.